marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/data-hub-http-user.json

```diff
@@ -0,0 +1,36 @@
+{
+  "role-name": "data-hub-http-user",
+  "description": "Contains a set of privileges needed for network access",
```

Comment:
For consistency with other DHF roles, the description should start with "Permits" - I think we should do "Permits use of HTTP functions" since it's the HTTP functions that users are after here.